src/iop/crop.c

```diff
@@ -1329,13 +1329,25 @@ void gui_post_expose(struct dt_iop_module_t *self, cairo_t *cr, int32_t width, i
     g_object_unref(layout);
   }
 
-  // draw crop area guides
-  dt_guides_draw(cr, g->clip_x * wd, g->clip_y * ht, g->clip_w * wd, g->clip_h * ht, zoom_scale);
-
   cairo_set_line_width(cr, DT_PIXEL_APPLY_DPI(2.0) / zoom_scale);
   dt_draw_set_color_overlay(cr, FALSE, 1.0);
```

Comment:
Now tested that PR and it's ok to me except that handle lines are too much visible (I know it's the same on master but the effect change as now they are always visible). I would suggest to just set that line to 0.5 instead of 1.0. Balance between handle and grid overlay set is better to me with 0.5 value.

Reply:
I did not test 0.5 as suggested by @Nilvus but testing last version I think the line are still too visible. So 0.5 is probably better. I'll check that tomorrow.

Reply:
Both parameters of `dt_draw_set_color_overlay` and `cairo_set_line_width` have an impact on visibility. An additional `cairo_set_line_width` could be added after `dt_guides_draw` to have a difference between the _active_ grab region and the rest of them. I'd say just play around with those settings and let me know what you want me to put in the final version. Or take this and make further adjustments afterwards.

Reply:
That's what I tested (both lines) and on my taste, best setting I find by now is what I suggest, only that change.

Reply:
Thanks. Let's wait until Pascal confirms he's happy with that too.